packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -128,23 +134,37 @@ function NetworkListbox({
       </Listbox.Button>
 
       <Listbox.Options className="absolute z-20 mt-2 rounded-xl bg-white shadow-[0px_4px_12px_#9e9e9e]">
-        {options.map((option, index) => (
-          <Listbox.Option
-            key={option.chainID}
-            value={option}
-            className={twMerge(
-              'flex h-12 cursor-pointer items-center space-x-2 px-4 hover:bg-blue-arbitrum hover:bg-[rgba(0,0,0,0.2)]',
-              getOptionClassName(index)
-            )}
-          >
-            <img
-              src={getOptionImageSrc(option)}
-              alt={`${getNetworkName(option.chainID)} logo`}
-              className="w-8"
-            />
-            <span>{getNetworkName(option.chainID)}</span>
-          </Listbox.Option>
-        ))}
+        {options.map((option, index) => {
+          return (
+            <Tooltip
+              key={option.chainID}
+              show={option.disabled}
+              content={option.disabledTooltip}
+              wrapperClassName="w-full"
+              theme="dark"
+            >
+              <Listbox.Option
+                value={option}
+                className={twMerge(
+                  'flex h-12 cursor-pointer items-center space-x-2 px-4 hover:bg-blue-arbitrum hover:bg-[rgba(0,0,0,0.2)]',
+                  getOptionClassName(index),
+                  option.disabled ? 'pointer-events-none opacity-40' : ''
+                )}
+                disabled={option.disabled}
+              >
+                <img
+                  src={getOptionImageSrc(option)}
+                  alt={`${getNetworkName(option.chainID)} logo`}
+                  className={twMerge(
+                    'max-h-10 w-8',
+                    isNetwork(option.chainID).isEthereum ? 'px-1' : ''
+                  )}
+                />
+                <span>{getNetworkName(option.chainID)}</span>
```

Comment:
Getting this weird text overflow...
![image](https://user-images.githubusercontent.com/7558499/198278815-47bb5e0f-c9f9-46ce-bb0f-611ff99ebf03.png)